packages/contracts/src/tx-util.ts

```diff
@@ -8,7 +8,7 @@ import {
   NonPayableTransactionObject,
   PayableTx,
   NonPayableTx,
-} from './contracts/types'
+} from '../src/contracts/types'
```

Comment:
let's revert this line ;)

Reply:
Updated at 33b23ae

Reply:
Maybe that commit is missing this file? would u plz check this once again?

Reply:
Forgot to revert this one.
resolved c5e2c52